### DISCUSSION TITLE
Windows Sandbox Crashing

### DISCUSSION THREAD

**Discussion by ghost** [gucci-on-fleek/lockdown-browser#31] [Q&A]
You wouldn't happen to know why every time that I open a Windows Sandbox session I am able open it for awhile, but I always get a force close with the associated message after a few minutes? Any help to remedy this problem would be very appreciated. Thank you very much for your project!

<img width="306" alt="Screenshot 2023-09-09 175817" src="https://github.com/gucci-on-fleek/lockdown-browser/assets/144567468/90f20726-7d6f-4811-b997-0138d12b710a">

**Comment by gucci-on-fleek** [comment]
Hmm, that's interesting. That specific error message means that something forcibly killed `VmComputeAgent.exe` from inside the _Sandbox_ (in my experience, at least). Now I'm pretty sure that you're not killing the process, so now we need to figure out what is killing it. Pretty much 3 options that I can think of:

1. Something on the host side is killing the process.

    Not really sure why this would happen, but perhaps some overzealous antivirus? Probably not this.

2. The _Browser_ is killing the process.

    The _Browser_ often tries to kill "unauthorized" processes, and I know that it specifically detects `VmComputeAgent.exe`. As far as I'm aware, the _Browser_ always prompts you before killing any processes on startup, and it never tries to kill processes after it's finished the startup process.

    Perhaps they've updated something lately to change this, but in that case I'd think that I would have seen more complaints from others.

3. Something else inside the _Sandbox_ is killing the process.

    I'm not really sure what else this could be. The startup script messes with `VmComputeAgent.exe`, but it does so right at the start, so you'd have the issue within seconds of starting. There's no antivirus or any other programs inside the _Sandbox_, so this would need to be something built in to Windows. Windows _shouldn't_ be doing anything like this, but bugs are definitely possible. 


None of these options really make any sense to me, so I'm not too sure what's going on.

Perhaps check the “Event Viewer”? You can open it by right-clicking on the Start Menu then clicking on the aptly-named option. Anything relevant will probably be in "Windows > System" or "Windows > Application", but there might be something inside "Applications and Services > … Hyper-V" (or something with a similar name). There will be _lots_ of stuff in these logs; you're only interested in things that mention "Hyper-V", "Sandbox", "Remote Desktop", or "Virtualisation" and happen within a few seconds of you seeing the dialog.

**Reply by ghost** [reply]
I am not exactly sure what the problem is. On my other machine it works perfectly fine and is completely stable.
